transmission_interface/test/simple_transmission_test.cpp

```diff
@@ -174,7 +174,8 @@ INSTANTIATE_TEST_CASE_P(
     SimpleTransmission(10.0, 1.0),
     SimpleTransmission(10.0, -1.0),
     SimpleTransmission(-10.0, 1.0),
-    SimpleTransmission(-10.0, -1.0)));
+    // cppcheck-suppress syntaxError
```

Comment:
please remove this.

It isn't related to fixing the thread, I'd really like to not have manual linter management in code

Reply:
please propose an alternative fix for this then. It does not compile/pass tests without this.

Reply:
https://github.com/ros-controls/ros2_control/pull/296 or https://github.com/ros-controls/ros2_control/pull/297, please pick

Reply:
done